src/Raven.Server/Utils/MicrosoftLogging/SparrowLoggerWrapper.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using Microsoft.Extensions.Logging;
+using Sparrow.Logging;
+
+namespace Raven.Server.Utils.MicrosoftLogging;
+
+public class SparrowLoggerWrapper<TLogger> : ILogger<TLogger>
+{
+    private readonly Logger _sparrowLogger;
+
+    public SparrowLoggerWrapper(Logger sparrowLogger)
+    {
+        _sparrowLogger = sparrowLogger;
+    }
+
+    public IDisposable BeginScope<TState>(TState state) where TState : notnull
+    {
+        return NullScope.Instance;
+    }
+
+    public bool IsEnabled(LogLevel logLevel) => true;
```

Comment:
are you sure changing the settings file will somehow magically disable some of the log levels?

Reply:
No, before that you have the check for the configuration.